src/main/java/controllers/GameCreationController.java

```diff
@@ -26,12 +26,17 @@ public GameCreationController(GameCreationInputBoundary inputBoundary) {
      * @param playersInfo The User requested Player names and if they are to be ComputerPlayers or not.
      * @return A GameCreationResponseModel detailing if a Game was created.
      */
-    public GameCreationResponseModel createGameResponse(ArrayList<PlayerCreationInformation> playersInfo) {
+    public boolean createGameResponse(ArrayList<PlayerCreationInformation> playersInfo) {
         HashMap<String, Boolean> requestInfo = new HashMap<>();
         for(PlayerCreationInformation info: playersInfo) {
             requestInfo.put(info.getPlayerName(), info.getIsComputerPlayer());
         }
+        // immediately return false if there are duplicate names
+        if(requestInfo.keySet().size() != playersInfo.size()) {
+            return false;
```

Comment:
Looks good in response to my earlier comment. However, as a follow up suggestion, perhaps notify users somehow why they cannot create the game? Otherwise, they would be stuck at the game creation page without knowing the reason.

Reply:
Ive added a message that should pop up when this happens.

Reply:
Awesome, seems I missed your updated version just by a bit!